File: Projeto02/AtividadeSLN/Atividade/Banco.py
class Banco:
    def __init__(self):
        self._nome = ""
        self._codigo_bacen = ""

    @property
    def nome(self):
        # Retorna o nome sempre em maiúsculas
        return self._nome.upper()

    @nome.setter
    def nome(self, value):
        self._nome = value

    @property
    def codigo_bacen(self):
        return self._codigo_bacen

    @codigo_bacen.setter
    def codigo_bacen(self, value):
        # Valida se não é nulo ou vazio
        if value and value.strip():
            self._codigo_bacen = value
        else:
            print("Erro: O Código BACEN não pode ser vazio.")

File: Projeto02/Projeto02 Fase03SLN/Projeto02 Fase03/Banco.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Projeto02
{
    internal class Banco
    {
        private string _nome;
        public string Nome
        {
            get { return _nome.ToUpper(); }
            set { _nome = value; }
        }


        private string _codigoBACEN;
        public string CodigoBACEN
        {
            get { return _codigoBACEN; }
            set
            {
                if (!string.IsNullOrEmpty(value))
                    _codigoBACEN = value;
                else
                    Console.WriteLine("Erro: O Código BACEN não pode ser vazio.");
            }
        }
    }
}